src/screens/Followings/elements/TabBar/FollowingTabBar.js

```diff
@@ -57,7 +58,7 @@ const S = StyleSheet.create({
   toptabcontainer: {
     flexDirection: 'row',
     backgroundColor: colors.white,
-    borderBottomColor: '#00000050',
+    borderBottomColor: COLORS.black30percent,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with dynamic colors from a theme object is a good practice for consistency and maintainability. However, make sure that the new color `COLORS.black30percent` matches the intended visual design as closely as possible to the old color `#00000050`.
```diff
-    borderBottomColor: '#00000050',
+    borderBottomColor: COLORS.black30percent,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->